### DISCUSSION TITLE
Have problem while Uploading proof of my academic status

### DISCUSSION THREAD

**Discussion by ezthor** [community/community#67955] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

After I upload my image, it shows "Thanks for uploading your photo. Please complete the form and submit when you're ready." ,however there is no bottom for me to select the proof type, and it shows an error message after I process my application :
"Your application cannot be reviewed until you fix the following:
1.You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
2.Please select proof type 'Other' for this image."
I have already completed my information(using my full name) but enable to select proof type because I can't find the bottom, both tried using my computer and mobile phone ,how can I fix it ?

**Comment by hjrPhoebus** [comment]
Have you solve this problem? i meet it too

**Reply by ezthor** [reply]
我确实解决了，但说实话我也不知道是怎么解决的，全部信息改了之后我刷新了几次没管它了，过了一阵子再提交谷歌翻译过的学信网学籍证明，它提示我这个已经提交过了（不能提交相同文件），我就用画笔在上面画了一下，然后成功提交了（因为没有秒发邮件告诉我未能通过），但是照样没有选择proof type，过了两三天告诉我已经成功申请了。总而言之，改完信息之后可以稍等一下，重启一下，可能就自然解决了。

**Reply by samus-aran** [reply]
Hi @hjrPhoebus 👋 

Thanks for raising this, we'll share this with the internal team. You shouldn't be penalised if you are unable to select the evidence type yourself. 🙂 I'll update this thread if there are any updates.

But I would suggest that you refrain from submitting multiple, repeated submissions without alterations as this may affect your application negatively.

**Reply by samus-aran** [reply]
Hi @hjrPhoebus 👋 

If you have been repetitively unsuccessful using a particular evidence type we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status. The most preferred are Student IDs.

Hope this helps 👍

**Comment by 0xTanzim** [comment]
Generally, it takes 7-10 days for you to become eligible. Please wait for a few more days and you will be eligible.

**Reply by Tanzimhossain222** [reply]
💕

**Comment by samus-aran** [comment]
Hi @ezthor 👋 

Seeing you have a the PRO badge the assumption that you were able to over come this? Would you be able to share with the community if you were able to see the evidence type form to select from?

**Reply by ezthor** [reply]
I did manage to submit my application successfully. However, I couldn't select the evidence type form till successfully submitting it. In my case, I was able to submit successfully without needing to select the evidence type form, although I'm not sure why.

**Comment by Momokozo** [comment, marked as answer]
Hey guys! Try this! CHANGE other proof documents, TRY student ID card! I haven't found the "proof type" form, and I tried to submit my verification report of student status, but I encountered the prompt asking me to select "other" proof type. I changed another proof document and luckily succeeded! Hope it helpful!

：换一个证明材料！学信网证明不行就换学生卡，中文的就手动翻译一下学生卡上的信息，然后两张图片叠在一起，手机拍个照！上传！就成功上传了！（应该可以解决这个问题：不显示“proof type”表单却叫你选“other”选项这个bug）

**Reply by 15935856121** [reply]
你好，我想问一下这个学生卡是指的是学校的一卡通嘛，俩张图片是那俩张呀

**Reply by mangoleaves** [reply]
A successful case from a Chinese student: I took a photo of my student ID card (even though it didn’t have any dates on it) and captured a screenshot of my student status verification report from xuexin.com. Then, I used Google Translate to translate them into English (by uploading the images). Afterward, I combined these images into a single file and uploaded it.
I received approval within a few minutes of uploading the image.
Hope this helps other students!
我拍了学生证的照片（上面没有日期），从学信网截图了学籍验证报告。然后用谷歌翻译把它们翻译成英文，（上传图片翻译可以在原文位置得到英文翻译结果）。最后把这些图片合并成一个图片上传。上传后几分钟就通过了。

**Comment by trooperken** [comment]
I also have an issue with proof of my academic status, but in my case they believe I'm a student.
However, this is not correct. I have submitted my faculty ID as proof of my academic status and have been working as a staff member for 20+ years. My name and profile are on the staff page of the school system website. I also receive the emails from GitHub Education on my staff email account.

I'd love to direct them to the staff page of the school system website or send them a snapshot of it but there doesn't seem to be any way?
Below is a screenshot with some information is 
![hcpss](https://github.com/community/community/assets/8365015/0545a137-c17f-463d-b747-3b0ce44ecbac)
redacted.

**Reply by samus-aran** [reply]
Hi @trooperken 👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Comment by ApocalCFH** [comment]


**Reply by ApocalCFH** [reply]
Photos that can only be taken with the front camera are too blurry and are always recognized as the same one
